src/components/Reactions/ReportActionItemEmojiReactions.js

```diff
@@ -44,27 +32,68 @@ const propTypes = {
 
 const defaultProps = {
     ...withCurrentUserPersonalDetailsDefaultProps,
+    emojiReactions: {},
 };
 
-function ReportActionItemReactions(props) {
+function ReportActionItemEmojiReactions(props) {
     const {reactionListRef} = useContext(ReportScreenContext);
     const popoverReactionListAnchor = useRef(null);
-    const reactionsWithCount = _.filter(props.reactions, (reaction) => reaction.users.length > 0);
+    let totalReactionCount = 0;
+
+    // Each emoji is sorted by the oldest timestamp of user reactions so that they will always appear in the same order for everyone
+    const sortedReactions = _.sortBy(props.emojiReactions, (emojiReaction, emojiName) => {
+        // Since the emojiName is only stored as the object key, when _.sortBy() runs, the object is converted to an array and the
+        // keys are lost. To keep from losing the emojiName, it's copied to the emojiReaction object.
+        // eslint-disable-next-line no-param-reassign
+        emojiReaction.emojiName = emojiName;
+        const oldestUserReactionTimestamp = _.chain(emojiReaction.users)
+            .reduce((allTimestampsArray, userData) => {
+                if (!userData) {
+                    return allTimestampsArray;
+                }
+                _.each(userData.skinTones, (createdAt) => {
+                    allTimestampsArray.push(createdAt);
+                });
+                return allTimestampsArray;
+            }, [])
+            .sort()
+            .first()
+            .value();
+
+        // Just in case two emojis have the same timestamp, also combine the timestamp with the
+        // emojiName so that the order will always be the same. Without this, the order can be pretty random
+        // and shift around a little bit.
+        return (oldestUserReactionTimestamp || emojiReaction.createdAt) + emojiName;
+    });
 
     return (
         <View
             ref={popoverReactionListAnchor}
             style={[styles.flexRow, styles.flexWrap, styles.gap1, styles.mt2]}
         >
-            {_.map(reactionsWithCount, (reaction) => {
-                const reactionCount = reaction.users.length;
-                const reactionUsers = _.map(reaction.users, (sender) => sender.accountID);
-                const emoji = _.find(emojis, (e) => e.name === reaction.emoji);
-                const emojiCodes = EmojiUtils.getUniqueEmojiCodes(emoji, reaction.users);
-                const hasUserReacted = Report.hasAccountIDReacted(props.currentUserPersonalDetails.accountID, reactionUsers);
+            {_.map(sortedReactions, (reaction) => {
+                const reactionEmojiName = reaction.emojiName;
+                const usersWithReactions = _.pick(reaction.users, _.identity);
+                let reactionCount = 0;
+                _.forEach(usersWithReactions, (user) => {
+                    if (user.skinTones) {
+                        reactionCount += _.size(user.skinTones);
+                    } else {
```

Comment:
`user.skinTones` sounds like an array or object. @stitesExpensify would you please add an example of the old and new emoji reactions formats to the PR description? I have forgotten what the formats look like.

Reply:
Updated the OP with the new and old format. I believe you're right that we can never not have a skintone, it will always have the object or be null, so we can just always count size here